### PR TITLE
Separate kinds for user emojis and emoji packs

### PR DESCRIPTION
**Pull request by verbiricha**

### CODE REVIEW

File: 51.md [hunk 1]
```diff
@@ -104,6 +105,10 @@ An event with kind `10000` is defined as a replaceable list event for listing co
 
 An event with kind `10001` is defined as a replaceable list event for listing content a user wants to pin. Any standardized tag can be included in a Pin List.
 
+### Custom Emoji List
+
+An event with kind `10030` is defined as a replaceable list event for listing [emoji](30.md) a user wants to reuse. Only 'emoji' tags can be included in a Custom Emoji List.
```
Comment (SnowCait) [inline 51.md]: >we could use kind 10030 as User Emoji List event, containing a list of emoji or a tags pointing to 30030 Custom Emoji Lists.

'a' tags pointing to 30030 may be allowed.

**Review by SnowCait — CHANGES_REQUESTED** [review]
Thank you!
I commented about 'a' tags.

File: 51.md (hunk 1 above)
Comment (verbiricha) [reply 51.md]: I see, misunderstood your comment. I've reverted the last commit so `a` refs to 30030 lists are allowed.

Comment (SnowCait) [reply 51.md]: Nice, perfect. 💯

**Pull request merged**